config.yaml

```
@@ -10,6 +10,7 @@ weather:
     threshold_wet: 2200
     threshold_rainy: 1800
     safety_delay: 15
+    ignore: None # can be a list, e.g. 'rain','cloud','gust','wind'
```

`ignore` is a little bit generic, maybe `ignore_unsafe`?

```suggestion
    ignore_unsafe: None # can be a list, e.g. 'rain','cloud','gust','wind'
```